packages/react-debug-tools/src/__tests__/ReactHooksInspectionIntegration-test.internal.js

```diff
@@ -212,6 +212,122 @@ describe('ReactHooksInspectionIntergration', () => {
     ]);
   });
 
+  describe('useDebugValue', () => {
```

Comment:
Should we also have a test for `useDebugValue` on top level of component? Outside a Hook.

Reply:
Also should we add tests for updates? To verify `inspectHooksOfFiber` can find latest version. (In case it matters — I don't understand this enough to tell yet.)